modules/library/main/src/test/java/org/geotools/styling/zoom/TestUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2016 Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2014-2016 Boundless Spatial
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.styling.zoom;
+
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.describedAs;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
+
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+public enum TestUtils {
+    ;
```

Comment:
Weird eh? Necessary semicolumn for an empty enum. I believe this was @smithkm attempt to ensure this class could not be instantiated.